internal/operator/operator.go

```diff
@@ -731,6 +734,7 @@ func (r *minioGroupBindingReconciler) Reconcile(ctx context.Context, req reconci
 			Members:  []string{gb.Status.CurrentSpec.User},
 			IsRemove: true,
 		})
+		err = ignoreMadminErrorCode(err, "XMinioAdminNoSuchUser")
```

Comment:
the fix for `MinioGroupBinding` deletion is to catch and ignore errors when the user (for the group attachment) no longer exists.